document-generation/generate-document.ts

```diff
@@ -33,6 +38,20 @@ import {
 } from "./handlebarUtils/helpers";
 
 async function baseHandler(event: RequestEvent<GenerateDocumentRequest>): Promise<ApiBase64SuccessResponse> {
+  const { documentType } = event.body;
+  switch (documentType) {
+    case DocumentType.DESCRIPTION_OF_WORK:
+      return generatePdf(event);
+    case DocumentType.INDEPENDENT_GOVERNMENT_COST_ESTIMATE:
+      return generateXlsx();
+    default:
+      return new ValidationErrorResponse(`Invalid document type: "${documentType}"`, {
+        cause: `Invalid document type "${documentType}" provided. Please provide a valid document  type.`,
+      });
+  }
+}
```

Comment:
If this grows in complexity, we may want to consider using Step Functions for this. I think the flows are also going to get more complicated over time.

`generatePdf` isn't necessarily true either -- it's specifically generating a "Description of Work" and would need to do that whether it was a PDF or a DOCX. Same for Xlsx, really. And if we had a separate document that used either data type, we'd probably rename the functions.

Anyway, not a big deal to me at the moment. Something to keep in mind for future expansion though. So not worth fixing unless you've gotta make changes for later review comments anyway.